.github/PULL_REQUEST_TEMPLATE/continuous-integration--ci.md

```diff
@@ -0,0 +1,115 @@
+---
+name: Continuous Integration
+about: Add to the CI of the project
+title: "ci(scope-of-PR): description of PR"
+labels: ''
+assignees: ''
+---
+
+# 🚩 Linked issue
+
+**Please remember to link the according issue here in this section with a [hyperlink](https://github.com/cuhacking/2025/issues).** This will give reviewers a chance to look to see what problem you are trying to solve.
+
+## 🛠 How does this PR address the issue(s)? How does it solve the problem?
+
+Showcase your implemented solution for the issue. Explain your additions/changes to the project
+
+## 🧠 Describe any alternatives you've considered
+
+Demonstrate why this is the correct/best solution specifically (in contrast to other solutions).
+
+## ✨ Dependency Changes (if applicable)
+
+If there are any dependency changes, list them here.
+
+## 📸 Before and After Screenshots (if applicable)
+
+|        Before         |        After         |
+| :-------------------: | :------------------: |
+| BEFORE IMAGES GO HERE | AFTER IMAGES GO HERE |
+
+## ✍️ Additional comments (if applicable)
+
+If there are any comments to add, list them here.
+
+# ✅ Checklist
+
+Make sure your PR follows the according checklist:
+
+### **Code Quality**
+
+<table>
```

Comment:
Very small nitpick, I don't really mind, but would like your opinion on this as well. We can use MD to write these:
Example:
- [ ] checkbox

I did this with "- [ ] checkbox"
Lmk your thoughts

Reply:
does not work in mdx...

Reply:
or at least you won't be able to interact with the checkbox at all

Reply:
Really, on my end I can interact with it?

Reply:
weird, for me it was not working nicely at all...

Reply:
How does it work in this comment?

Reply:
yes, but mdx is not rly md, i talked to farabi abt it and he said it was normal

Reply:
Yea, i'm aware, but from waht I know checkboxes work in MDX and MD. 

Refer to the PR below and you can interact with the checkboxes: 
PR #60 is a good example